src/System.Drawing.Primitives/src/System/Drawing/KnownColorTable.cs

```diff
@@ -0,0 +1,463 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Drawing
+{
+    static internal class KnownColorTable
```

Comment:
guidelines: rule 5 https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md

visibility should be first. so "internal static class"